zio/ndjsonio/parser.go

```diff
@@ -77,6 +77,10 @@ func (*stubTypeOf) Coerce(zv zcode.Bytes, typ zng.Type) zcode.Bytes {
 	return nil
 }
 
+func (*stubTypeOf) Id() int {
```

Comment:
Nit that can wait for another PR: I like ID better than Id.

Reply:
Ok, what should I call zng.TypeRecord.ID?  same for set, vector

Reply:
How about id? Its only assignments are in package zng, so it needn't be public.